icu4c/source/common/common.vcxproj

```diff
@@ -91,6 +121,17 @@
       <ImportLibrary>..\..\$(IcuLibOutputDir)\icuuc.lib</ImportLibrary>
     </Link>
   </ItemDefinitionGroup>
+  <ItemDefinitionGroup Condition="'$(Configuration)'=='ReleaseStatic'">
+    <ClCompile>
+      <RuntimeLibrary>MultiThreaded</RuntimeLibrary>
+      <FunctionLevelLinking>true</FunctionLevelLinking>
+    </ClCompile>
+    <Link>
+      <OutputFile>..\..\$(IcuBinOutputDir)\icuuc67.dll</OutputFile>
```

Comment:
If this is supposed to be building a static library, why is the `OutputFile` a DLL?
(IIRC, I think the OutputFile line just is removed for Static Libraries).